contracts/HubPool.sol

```diff
@@ -556,6 +556,10 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         ancillaryData = AncillaryData.appendKeyValueAddress(ancillaryData, "proposer", rootBundleProposal.proposer);
     }
 
+    // This function allows a caller to load the contract with raw ETH to perform L2 calls. This is needed for arbitrum
+    // calls, but may also be needed for others.
+    function loadEthForL2Calls() public payable {}
```

Comment:
so ugly, but neccessary. Alternatively, why not make the functions that need to pay ETH for arbitrum messages `payable`? I'm guessing its only the functions that call `adapter.relayMessage`?

Reply:
As discussed IRL, we can decide if we want to address this in a follow-up, but not a super high priority, as doing it this way may end up being more complicated.